### DISCUSSION TITLE
Отображение имени поля при скролле в перевернутой таблице

### DISCUSSION THREAD

**Discussion by qye1** [totumonline/totum-ru-issues-and-discussions#337] [Есть идея, а что если...]
Можно ли отображать имя поля при скролле перевернутой таблиц так, как это реализовано в неперевернутой таблице?

![2023-03-21_16-27-04](https://user-images.githubusercontent.com/70027355/226620225-6c030374-766a-4688-9e30-a835900b31e5.png)
![2023-03-21_16-21-06](https://user-images.githubusercontent.com/70027355/226619654-9afd2f7c-b9f5-4f83-a734-df30c3b058a6.png)

**Comment by ttm-alexeypolunin** [comment]
Если имеются в виду `id` и они сейчас не отображаются поверх, значит не сделали.

Запишем. Сроки реализации неизвестны.

Если прям очень нужно и без этого миллионы не генерятся, то срочно напрячь разработку такой задачей = 120 € (с документами 180 €).

**Reply by ttm-alexeypolunin** [reply]
Если дело не в накладывании при прокрутке, а в том, что бы покозать что-то человеческое вместо `id` вот так:

<img width="803" alt="Снимок экрана 2023-03-21 в 17 25 21" src="https://user-images.githubusercontent.com/55755565/226637116-1c2cd12a-72c8-418d-b618-9b223384ac85.png">

То надо задать параметр таблицы (в настройках таблицы) — `Основное поле`
